analyses/org.eclipse.tracecompass.incubator.executioncomparision.core/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,34 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %Bundle-Name
+Bundle-Vendor: %Bundle-Vendor
+Bundle-SymbolicName: org.eclipse.tracecompass.incubator.executioncomparision.core;singleton:=true
```

Comment:
Ideally, a refactor to rename this package to executioncomparison should be done.

Reply:
Done